pkg/apis/imageregistry/v1/types.go

```diff
@@ -77,63 +172,33 @@ type ImageRegistryConfigStorage struct {
 	Swift      *ImageRegistryConfigStorageSwift      `json:"swift,omitempty"`
 }
 
-type ImageRegistryConfigRequestsLimits struct {
-	MaxRunning     int           `json:"maxrunning,omitempty"`
-	MaxInQueue     int           `json:"maxinqueue,omitempty"`
-	MaxWaitInQueue time.Duration `json:"maxwaitinqueue,omitempty"`
-}
-
 type ImageRegistryConfigRequests struct {
 	Read  ImageRegistryConfigRequestsLimits `json:"read,omitempty"`
 	Write ImageRegistryConfigRequestsLimits `json:"write,omitempty"`
 }
 
-type ImageRegistryConfigRoute struct {
-	Name       string `json:"name"`
-	Hostname   string `json:"hostname"`
-	SecretName string `json:"secretName"`
-}
+type ImageRegistryConfigRequestsLimits struct {
 
-type ImageRegistrySpec struct {
-	ManagementState operatorsv1api.ManagementState `json:"managementState"`
-	HTTPSecret      string                         `json:"httpSecret,omitempty"`
-	Proxy           ImageRegistryConfigProxy       `json:"proxy,omitempty"`
-	Storage         ImageRegistryConfigStorage     `json:"storage,omitempty"`
-	Requests        ImageRegistryConfigRequests    `json:"requests,omitempty"`
-	TLS             bool                           `json:"tls,omitempty"`
-	CAConfigName    string                         `json:"caConfigName,omitempty"`
-	DefaultRoute    bool                           `json:"defaultRoute,omitempty"`
-	Routes          []ImageRegistryConfigRoute     `json:"routes,omitempty"`
-	Replicas        int32                          `json:"replicas,omitempty"`
-	LogLevel        int64                          `json:"logging,omitempty"`
-}
+	// MaxRunning sets the maximum in flight api requests to the registry
+	MaxRunning int `json:"maxrunning,omitempty"`
```

Comment:
Are we not doing CamelCase for these for the json? maxrunning vs maxRunning (same for the other two below)

Reply:
they should have been.  updated.